Update harness-platform-faqs.md (#6523)
harness/developer-hub: kb/platform/harness-platform-faqs.md

This limitation is by design to uphold security protocols. Delegates often operate with scoped permissions at the organization or project level, and allowing access to the account's private key via a delegate would pose a significant security risk. Therefore, regardless of the delegate's ability to communicate with Harness Manager, access to the account's private key is strictly controlled and not available through delegate operations.

### Does Harness SAML work with any SSO provider?

> Yes, Harness provides support for OKTA and Azure Onelogin out of the box, but you can add any custom SSO with Harness. All you need is the Harness SAML endpoint URL and SAML metadata file. 

Reviewer: ```suggestion
Yes, Harness provides support for Okta and Azure Onelogin out of the box, but you can add any custom SSO with Harness. All you need is the Harness SAML endpoint URL and SAML metadata file. 
```

For SAMl with Microsoft Entra ID (formerly Azure), under **Enterprise Applications**, select **SSO Settings**, then select **Edit User and Group properties**. Edit the **Group Claim** and validate that the attribute is set to Group ID. If it is set to any other attribute, you must provide that value when linking the Harness user group with the Microsoft Entra ID group using the authorization.

### When creating multiple Harness apps in Microsoft Entra ID, why can't I use the app.harness.io entity ID?

The entity is unique for each application. You can use any entity ID and add it in the Harness UI when setting up SSO.

### When we have a user group established through SCIM and others connected via SSO within the same IDP, how do both methods populate the users?

When a user group is created through SCIM, it synchronizes and directly creates the user groups and users within the IDP as they exist. However, for SSO-linked user groups, users are populated only when they sign in to Harness using SAML.

> ### We removed users from a user group in our SSO app, why does the Harness UI still show the users? 

Reviewer: ```suggestion
### We removed users from a user group in our SSO app, why are they still in the Harness UI? 
```

"Exit code 137" typically indicates an out-of-memory error. When a process in a system exhausts its allocated memory resources, the operating system sends a termination signal to the process. In the case of "Exit code 137," this signal signifies that the process was terminated due to running out of memory. This error commonly occurs when a program or container attempts to allocate more memory than is available, leading to termination by the system to prevent resource exhaustion and potential system instability.

> ### Can multiple IDP work even when the IDP are different like Azure and OKTA?

Reviewer: ```suggestion
### Do multiple IDPs work even when the IDPs are different like Microsoft Entra ID and Okta?
```

Yes Once the FF PL_ENABLE_MULTIPLE_IDP_SUPPORT is enabled for your account you can configure multiple IDP it can either multiple IDP of OKta , Azure etc or all different like 1 of OKTA, other of Azure. 

> Once you click on Single Sign on it asks the User which IDP to select for login and then redirects based on that. 

Reviewer: ```suggestion
Upon enabling the feature flag `PL_ENABLE_MULTIPLE_IDP_SUPPORT` for your account, you gain the ability to configure multiple IDPs, which can include various combinations such as multiple instances of Okta, Microsoft Entra ID, or a mix of different providers like Okta and Microsoft Entra ID. When you select Single Sign-On, the system prompts you to select your preferred IDP for login, subsequently redirecting you based on your choice.
```
